test/calendar.ts

```diff
@@ -20,14 +15,26 @@ describe('calendar', function () {
     });
   }
 
-  //wait until the event is loaded on the calendar
+  // Not a pretty way to get events from currently used calendar control. but it's working.
+  function buildGetVisibleCalendarEvent(eventId: number) {
+    return `return testGetVisibleCalendarEvent(${eventId});`
+  }
+
+  async function getVisibleCalendarEvent(eventId: number): Promise<any> {
+    const event = await grist.executeScriptOnCustomWidget(buildGetVisibleCalendarEvent(eventId));
```

Comment:
Somehow feels like whoever wrote `executeScriptOnCustomWidget` went against the grain, forcing functions to be expressed as strings. This is uglier than any custom widget testing in grist-core.